src/pages-and-resources/discussions/messages.js

```diff
@@ -1,20 +1,12 @@
 import { defineMessages } from '@edx/frontend-platform/i18n';
 
 const messages = defineMessages({
-  configureApp: {
-    id: 'authoring.discussions.configure.app',
-    defaultMessage: 'Configure {name}',
-  },
   configure: {
     id: 'authoring.discussions.configure',
     defaultMessage: 'Configure Discussions',
   },
-  appLogo: {
```

Comment:
Note that appLogo and configModalTitle were both not being used, so they've just been removed.